docker-compose.yml

```diff
@@ -7,6 +7,8 @@ services:
   oereb-db:
     build: test-db
     image: camptocamp/oereb-db-dev
+    volumes:
```

Comment:
Why do we need this change (for this PR)?

Reply:
was only for debug, I will remove it